### PR TITLE
Make hits optional

### PR DESCRIPTION
**Pull request by danielmitterdorfer**
With this commit we ensure that statistics get still derived properly
even if there are no hits.

### CODE REVIEW

File: eventdata/runners/kibana_runner.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def kibana(es, params):
     sum_hits = 0
     max_took = 0
     for r in result["responses"]:
-        hits = r["hits"]["total"]
+        hits = r.get("hits", {}).get("total", 0)
```
Comment (drawlerr) [inline eventdata/runners/kibana_runner.py]: Perhaps the handling for hits-as-a-dict should be moved into the isinstance(hits, dict) block below?

Comment (danielmitterdorfer) [reply eventdata/runners/kibana_runner.py]: `hits` is always a dict, it's that `total` can either be a dict or an int?

Comment (dliappis) [reply eventdata/runners/kibana_runner.py]: Since hits is always a dict, maybe the conditional below (line 60) isn't needed?

Comment (danielmitterdorfer) [reply eventdata/runners/kibana_runner.py]: Ok, to be more explicit: `r["hits"]` is always a `dict`but not the variable `hits` (maybe we should provide a different name...)

Comment (dliappis) [reply eventdata/runners/kibana_runner.py]: Ah! thanks for the explanation.

**Comment by danielmitterdorfer** [comment]
thanks for the review both! :)

**Comment by danielmitterdorfer** [comment]
I've raised https://github.com/elastic/rally-eventdata-track/pull/79 now to improve test coverage.

**Pull request merged**